common/txmgr/types/fee_estimator.go

```diff
@@ -23,7 +23,7 @@ type Fee fmt.Stringer
 //go:generate mockery --quiet --name PriorAttempt --output ./mocks/ --case=underscore
 type PriorAttempt[F Fee, TX_HASH types.Hashable] interface {
 	Fee() F
-	GetChainSpecificGasLimit() uint32
+	GetChainSpecificFeeLimit() uint32
```

Comment:
Would now be a good opportunity to drop the `Get` prefix?
```suggestion
	ChainSpecificFeeLimit() uint32
```

Reply:
Actually, could we drop `ChainSpecific` too? That might have broader implications.
```suggestion
	FeeLimit() uint32
```

Reply:
> Would now be a good opportunity to drop the Get prefix?
I think we could drop the `Get` methods entirely and just access the struct directly since it's now generic...is that what you are suggesting? right now it's a bit of a mix of accessing directly or via the get methods

> Actually, could we drop ChainSpecific too? That might have broader implications.
i'd be hesitant to change the naming that far - I'm not sure if there was a reason to make it more specific 🤷🏼 since it's different than the top level `Tx.FeeLimit` naming

Reply:
Oh if the `Get`s are just distinguishing from internal fields of the same name, then nbd.

I thought `ChainSpecific` might just be a naming artifact from the old config interfaces but no need to make a mess now :shrug:

Reply:
let me see if i can standardize around a method - since those fields aren't internal. IIRC the `Get` methods were used as a bridge before the `Tx` and `TxAttempt` structs were generalized

Reply:
ok - this change is starting to spiral a bit with removing extraneous interfaces
I've captured it in a ticket to revisit: https://smartcontract-it.atlassian.net/browse/BCI-1230